spec/turns_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/card'
+require './lib/turns'
+
+
+RSpec.describe Card do
+  it 'exists' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+
+    expect(card).to be_instance_of(Card)
+  end
+end
+
+RSpec.describe Turns do
+  it 'exists' do
+    turns = Turns.new("Juneau", @card)
+
+    expect(turns).to be_instance_of(Turns)
+  end
+
+  it 'returns the card' do
+    turns = Turns.new("Juneau", @card)
+
+    expect(turns.card).to eq(@card)
+  end
+
+  it 'is correct' do
+    turns = Turns.new("Juneau", @card)
+    expect(turns.correct?).to eq(true)
+
+  end
+
+  it 'has feedback' do
+    turns = Turns.new("Juneau", @card)
```

Comment:
Same as above - use `card` since this instance variable has no context here.

I think you meant to use the `card` listed on line 7 of this file - it needs to be included inside of each `it` block for this to work. Right now, this test is failing, along with the one on line 28, for the same reason.